src/components/DownloadComponents/DownloadToggle/index.tsx

```diff
@@ -19,6 +19,7 @@ const DownloadToggle = ({
     [styles.active]: selected === 'CURRENT',
   });
 
+  const intl = useIntl();
```

Comment:
Could we use `injectIntl` instead of `useIntl`

We're trying to use HOC (High-order-components) instead of Hooks here for React-Intl, as it will wrap the component with `intl`.

(Just search for `injectIntl` in the codebase, and you will see how to use it 😄)

Reply:
@ovflowd Please Review below PR.

https://github.com/nodejs/nodejs.dev/pull/2935

Thanks!

Reply:
Absolutely, I'll update it!

Reply:
> @ovflowd Please Review below PR.

> https://github.com/nodejs/nodejs.dev/pull/2935

> Thanks!

Not sure what this comment is supposed to be.